package.json

```diff
@@ -18,7 +20,10 @@
     "react-router": "^6.4.4"
   },
   "devDependencies": {
+    "autoprefixer": "^10.4.13",
     "eslint": "8.27.0",
-    "eslint-config-next": "13.0.4"
+    "eslint-config-next": "13.0.4",
+    "postcss": "^8.4.21",
+    "tailwindcss": "^3.2.7"
   }
```

Comment:
If any new dependency gets added, make sure to push the `package.lock.json` file as well.